content/solids/functions.ts

```diff
@@ -321,6 +322,11 @@ export function honeycombIntro($step: Step) {
   });
 }
 
+export function netsProperties($step: Step) {
+  const $swipe = $step.$('x-binary-swipe') as BinarySwipe;
+  $swipe.on('complete', () => $step.score('all-swiped'));
+}
```

Comment:
This is ok for now, but we should try to automatically generate a goal for EVERY card in the swipe stack, and the restore all cards that have previously been correctly allocated when you refresh.